tvx/lotus/driver.go

```diff
@@ -42,6 +43,13 @@ func (d *Driver) ExecuteMessage(msg *types.Message, preroot cid.Cid, bs blocksto
 	if err != nil {
 		return nil, cid.Undef, err
 	}
+	// need to modify the VM invoker to add the puppet actor
```

Comment:
Adding the puppet actor to the VM, so that we can test messages that would otherwise fail preconditions.